deps.edn

```diff
@@ -193,23 +192,26 @@
   ;;    (dev/start!)
   :dev
   {:extra-deps
-   {clj-http-fake/clj-http-fake  {:mvn/version "1.0.3"
+   {
```

Comment:
for the main deps above we have the `{` on the same line as the first dependency